static/app/views/performance/newTraceDetails/trace.tsx

```diff
@@ -165,6 +166,7 @@ function Trace({
   const api = useApi();
   const {projects} = useProjects();
   const organization = useOrganization();
+  const location = useLocation();
```

Comment:
We shouldnt use this hook, it is problematic because it causes unnecessary rerenders everywhere.